dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -184,7 +184,40 @@ metrics-jmx/4.2.2//metrics-jmx-4.2.2.jar
 metrics-json/4.2.2//metrics-json-4.2.2.jar
 metrics-jvm/4.2.2//metrics-jvm-4.2.2.jar
 minlog/1.3.0//minlog-1.3.0.jar
-netty-all/4.1.68.Final//netty-all-4.1.68.Final.jar
+netty-all/4.1.72.Final//netty-all-4.1.72.Final.jar
```

Comment:
I see, so netty-all no longer has the contents of all the JARs below, so it's not redundant? that is OK if so. We could even exclude probably lots of these (SMTP? SOCKS?). Not necessary but might be nice if they're largeish and unused.

BTW the description says this updates log4j, but, log4j 2 is not in the build before or after this

Reply:
OK, let me try to clean up unnecessary Netty submodules

> BTW the description says this updates log4j, but, log4j 2 is not in the build before or after this

I have deleted the relevant description. As @attilapiros @attilapiros said, the upgrade of log4j2 in Netty does not affect Spark

Reply:
@srowen @attilapiros  e34a2c9 I clean up 28 Netty sub-modules that seem not to be used by spark, waiting for CI to test. I will also test these locally with Maven.